src/samples/FreezeOpenChannelSample.js

```diff
@@ -0,0 +1,582 @@
+                const updatedMessages = [...messages, message];
+                updateState({ ...state, messages: updatedMessages, messageInputValue: "" });
+
+            }).onFailed((error) => {
+                console.log(error)
+                console.log("failed")
+            });
+
+        }
+    }
+
+    const onFileInputChange = async (e) => {
+        if (e.currentTarget.files && e.currentTarget.files.length > 0) {
+            const { currentlyJoinedChannel, messages } = state;
+            const fileMessageParams = new FileMessageParams();
+            fileMessageParams.file = e.currentTarget.files[0];
+            currentlyJoinedChannel.sendFileMessage(fileMessageParams).onSucceeded((message) => {
+                const updatedMessages = [...messages, message];
+                updateState({ ...state, messages: updatedMessages, messageInputValue: "", file: null });
+
+            }).onFailed((error) => {
+                console.log(error)
+                console.log("failed")
+            });
+        }
+    }
+
+    const handleDeleteMessage = async (messageToDelete) => {
+        const { currentlyJoinedChannel, messages } = state;
+        await deleteMessage(currentlyJoinedChannel, messageToDelete);
+        const updatedMessages = messages.filter((message) => {
+            return message.messageId !== messageToDelete.messageId;
+        });
+        updateState({ ...state, messages: updatedMessages });
+
+    }
+
+    const updateMessage = async (message) => {
+        updateState({ ...state, messageToUpdate: message, messageInputValue: message.message });
+    }
+
+    const setupUser = async () => {
+        const { userNameInputValue, userIdInputValue } = state;
+        const sendbirdChat = await SendbirdChat.init({
+            appId: SENDBIRD_INFO.appId,
+            localCacheEnabled: false,
+            modules: [new OpenChannelModule()]
+        });
+
+
+
+        await sendbirdChat.connect(userIdInputValue);
+        await sendbirdChat.setChannelInvitationPreference(true);
+
+        const userUpdateParams = new UserUpdateParams();
+        userUpdateParams.nickname = userNameInputValue;
+        userUpdateParams.userId = userIdInputValue;
+        await sendbirdChat.updateCurrentUserInfo(userUpdateParams);
+
+        sb = sendbirdChat;
+        updateState({ ...state, loading: true });
+        const [channels, error] = await loadChannels();
+        if (error) {
+            return onError(error);
+        }
+        updateState({ ...state, channels: channels, loading: false, settingUpUser: false });
+    }
+
+    if (state.loading) {
+        return <div>Loading...</div>
+    }
+
+    if (state.error) {
+        return <div className="error">{state.error} check console for more information.</div>
+    }
+
+    console.log('- - - - State object very useful for debugging - - - -');
+    console.log(state);
+
+    const displayFrozenToastMessage = () => {
```

Comment:
Same messages as here. Also not too sure on the use of Toast here. Not sure everyone would get that. can we simplify?